src/hyperion/external_interaction/ispyb/ispyb_store.py

```diff
@@ -51,51 +51,61 @@ def experiment_type(self) -> str:
     def begin_deposition(
         self,
         data_collection_group_info: DataCollectionGroupInfo,
-        scan_data_info: ScanDataInfo,
+        scan_data_infos: Sequence[ScanDataInfo],
     ) -> IspybIds:
         ispyb_ids = IspybIds()
-        if scan_data_info.data_collection_info:
-            ispyb_ids.data_collection_group_id = (
-                scan_data_info.data_collection_info.parent_id
-            )
+        if scan_data_infos[0].data_collection_info:
+            ispyb_ids.data_collection_group_id = scan_data_infos[
+                0
+            ].data_collection_info.parent_id
 
         return self._begin_or_update_deposition(
-            ispyb_ids, data_collection_group_info, [scan_data_info]
+            ispyb_ids, data_collection_group_info, scan_data_infos
         )
 
     def update_deposition(
         self,
         ispyb_ids,
-        data_collection_group_info: DataCollectionGroupInfo,
+        data_collection_group_info: Optional[DataCollectionGroupInfo],
         scan_data_infos: Sequence[ScanDataInfo],
-    ):
+    ) -> IspybIds:
         assert (
             ispyb_ids.data_collection_group_id
         ), "Attempted to store scan data without a collection group"
         assert (
             ispyb_ids.data_collection_ids
         ), "Attempted to store scan data without a collection"
+        print(f"OBJECT PASSED IN IS {ispyb_ids}")
```

Comment:
Should: I assume this is for debugging? Can you remove please? Same with other places in this file.

Reply:
Doh